src/pool/srv_pool.c

```diff
@@ -1086,8 +1286,18 @@ pool_svc_step_up_cb(struct ds_rsvc *rsvc)
 			D_ERROR(DF_UUID": failed to read pool map buffer: "
 				""DF_RC"\n", DP_UUID(svc->ps_uuid), DP_RC(rc));
 		}
+		D_GOTO(unlock, rc);
+	}
+
+	prop_bits = DAOS_PO_QUERY_PROP_ALL;
+	rc = pool_prop_read(&tx, svc, prop_bits, &prop);
+	if (rc != 0) {
+		D_ERROR(DF_UUID": cannot get access data for pool, "
+			"rc="DF_RC"\n", DP_UUID(svc->ps_uuid), DP_RC(rc));
+		D_GOTO(unlock, rc);
 	}
 
```

Comment:
And this blank line should be removed too. Thanks a lot.

Reply:
ok